tools/ml-audio-content-moderation/terraform/terraform.tfvars

```diff
@@ -0,0 +1,7 @@
+project_id                  = ""
```

Comment:
You shouldn't commit the tfvars file. Instead, encourage customers to create their own and/or copy a `terraform.tfvars.sample`.

Reply:
LMK if I did it correct!